app/Repositories/CategoryRepository.php

```diff
@@ -10,8 +10,22 @@ class CategoryRepository
 
     protected $model;
 
-    public function __construct(Category $category)
-    {
+    public function __construct(Category $category){
         $this->model = $category;
     }
+
+    public function createCategory($categoryNew){
+        $id = $this->model->create($categoryNew)->id;
+        return $id;
+    }
+
+    public function updateCategory($id, $categoryNew){
+        $this->model->where('id', $id)->update($categoryNew);
```

Comment:
chỗ này chỉ cần `$this->update($id, $categoryNew)`

Reply:
trong lớp `CategoryRepositories` đang uses thằng `BaseRespositories` mà thằng `BaseRespositories` có hàm `update` nên có thể dùng của nó luôn ko cần viết thêm cái hàm `updateCategory`